library/firewall_lib.py

```diff
@@ -173,6 +184,49 @@
 except ImportError:
     HAS_FIREWALLD = False
 
+CONFIGS_TO_DELETE = [
+    "/etc/firewalld/zones/*",
+    "/etc/firewalld/policies/*",
+    "/etc/firewalld/direct.xml",
+]
+
+
+def hash_file(file_name):
+    out = check_output(["xmllint", "--c14n", file_name])
+    # return hashlib.md5(out).hexdigest()
+    return file_name
```

Comment:
We need to use the hash to compare the files.

Also - how to handle firewalld.conf?